human_pose_ros/Skeleton[] skeletons
int8 tracked_person_id
